mix.exs

```diff
@@ -37,7 +42,9 @@ defmodule Feedback.Mixfile do
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"}]
+     {:cowboy, "~> 1.0"},
+     {:comeonin, "~> 2.0"},
```

Comment:
using it in a subsequent PR https://github.com/dwyl/feedback/pull/33/files#diff-651f440688d3528ab409efd7e368c15fR37